### ISSUE TITLE
Retraction issue with the Pumps which renders them unusable (oil pumps)

### ISSUE THREAD

**Issue by Alastors** [GTNewHorizons/GT-New-Horizons-Modpack#7526]
#### Which modpack version are you using?
2.1.0.3a
#
#### If in multiplayer; On which server does this happen? 
Mine
#
#### What did you try to do, and what did you expect to happen?
Use a pump like normal
#
#### What happened instead? (Attach screenshots if needed)
okay so basically whenever a pump uses all of its internal inventory pipes and has none left it instantly recoils and becomes unusable until you break and replace (of which you have to do without powering so you can add the pipes or it will become unusable again). 

tl;dr: when a pump has no fluid pipes in them they stop working completely and pull pipes back into them
#
#### What do you suggest instead/what changes do you propose?

fix it so that it works like it used to, idk man

**Comment by Prometheus0000** [comment]
Probably related to https://github.com/GTNewHorizons/GT5-Unofficial/pull/388 @Glease

**Comment by Glease** [comment]
You right click it with a soft mallet to turn it back on. You can supply more than one stack of pipes with e.g. a hopper.

The autoretract is a requested feature. The original intention was to leave it unretracted until you manually tell it to retract by turning it off, but later people says it should be the otherwise, i.e. what it is now.

**Comment by Prometheus0000** [comment]
Shouldn't it only retract when there's no more fluid to get though, not just when you run out of pipes?

**Comment by Alastors** [comment]
thats what I said, thats why I added it again

**Comment by Glease** [comment]
Now that sounds a lot more reasonable. I was thinking you want to get rid of the pipe retraction feature as in _fix it so that it works like it used to_

**Comment by Alastors** [comment]
I do apologize for duping it,

**Comment by Alastors** [comment]
sorry didnt mean to do that

**Comment by Alastors** [comment]
oh no, I hated having to mine it down to the bottom, the retraction feature is a godsend, but automatically retracting when it runs out of pipes instead of pausing is kind of an issue
